src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -321,7 +321,7 @@ function ClimaLand.Canopy.set_canopy_prescribed_field!(
     t,
 ) where {FT}
     (; LAIfunction, SAI, RAI) = component.parameters.ai_parameterization
-    evaluate!(p.canopy.hydraulics.area_index.leaf, LAIfunction, t)
+    evaluate!(p.canopy.hydraulics.area_index.leaf, LAIfunction, floor(t))
```

Comment:
why do we need this here but not in for drivers?

Reply:
trying without...

Reply:
did it work without the floor?

Reply:
No :/ Im not sure. TimeVaryingInputs work fine in the site level runs, and work fine in global runs except here. I can look into it but would prefer to open an issue and tackle separately, if that is OK. (even next week, so I dont forget about it)

Reply:
What's the issue?

Reply:
@juliasloan25 Gabriele found the bug, it is in Clima Utilities, where we convert the simulation time into seconds. We must have lucked out before that dt*gamma was always an integer number of seconds, now with ARS343 that is not the case for the global run at least.

The atmos `evaluate!` commands may not trigger the error because (1) they are called only when we update via the callback, which is every 3 hours (integer number of seconds from the start time) or maybe because (2) the LAI data is coming from MODIS, not era5, and so may have a different start time in the file...or another reason TBD


Gabriele will open an issue in ClimaUtilities

Reply:
https://github.com/CliMA/ClimaUtilities.jl/issues/70

I think I removed the floor when porting to ClimaUtilities. We should find the best way to deal with this

Reply:
Yes, I think the parameters gamma and a42 may be the culprit? https://github.com/CliMA/ClimaTimeSteppers.jl/blob/d10a7d0a55d6edd8f21f6e5852beb54ec416c08d/src/solvers/imex_tableaus.jl#L200

vs RK4: https://github.com/CliMA/ClimaTimeSteppers.jl/blob/d10a7d0a55d6edd8f21f6e5852beb54ec416c08d/src/solvers/explicit_tableaus.jl#L90

which didnt require this (before I rebased the PR for the implicit solver changes)

Reply:
Interesting! Thank you for tracking this down :)